src/main/java/org/jmeterplugins/repository/PluginManagerCMD.java

```diff
@@ -33,6 +34,9 @@ protected int processParams(ListIterator listIterator) throws UnsupportedOperati
                 case "install-all-except":
                     installAll(listIterator, true);
                     break;
+                case "install-plugins":
```

Comment:
Let's use `install-for-jmx` to be more specific with the actual meaning of command